tests/model/test_spectrum.py

```diff
@@ -0,0 +1,18 @@
+from __future__ import absolute_import, division, print_function
+
+import pytest
+
+from dxtbx.model import Spectrum
+from scitbx.array_family import flex
+
+
+def test_spectrum():
```

Comment:
Belt & braces (and this is from experience of stupid errors in the past) - should probably have tests which exercise the code at 0.66Å and 1.5Å as well... 1Å is a dangerous wavelength for testing as `12398.4187/ƛ` == `12398.4187 * ƛ` 🙂